extensions/amp-story/1.0/page-advancement.js

```diff
@@ -546,16 +546,18 @@ export class ManualAdvancement extends AdvancementConfig {
   }
 
   /**
-   * Checks if click was inside of one of the side edges of the screen.
+   * Checks if click was inside of one of the side edges of the page.
    * @param {!Event} event
    * @param {!ClientRect} pageRect
```

Comment:
If pageRect is not needed since you're reading from the storeService, you can remove this param.

Reply:
Changed this bc offsetLeft & Top just weren't working on loading pages.